Source/SwiftLintFramework/Rules/Style/ConditionalReturnsOnNewlineRule.swift

```diff
@@ -18,33 +17,81 @@ public struct ConditionalReturnsOnNewlineRule: ConfigurationProviderRule, OptInR
             Example("if true,\n let x = true else {\n return true\n}"),
             Example("if textField.returnKeyType == .Next {"),
             Example("if true { // return }"),
-            Example("/*if true { */ return }")
+            Example("/*if true { */ return }"),
```

Comment:
Can this even be parsed correctly? I guess this example does not even reach the visitor.